ee/debug/checkups/power_windows.go

```diff
@@ -73,6 +73,23 @@ func (p *powerCheckup) Run(ctx context.Context, extraWriter io.Writer) error {
 		return fmt.Errorf("writing available sleep states output file: %w", err)
 	}
 
+	eventFilter := `Get-Winevent -FilterHashtable @{LogName='System'; ProviderName='Microsoft-Windows-Power-Troubleshooter','Microsoft-Windows-Kernel-Power'} -MaxEvents 500 | Select-Object @{name='Time'; expression={$_.TimeCreated.ToString("O")}},Id,LogName,ProviderName,Message,TimeCreated | ConvertTo-Json`
+	getWinEventCmd, err := allowedcmd.Powershell(ctx, eventFilter)
+	if err != nil {
+		return fmt.Errorf("creating powershell get-winevent command: %w", err)
```

Comment:
I will approve, but I wonder about creating the outfile first, and writing the errors into it. Otherwise, where will they go, and how do you see them reading the flare?

Reply:
oop sorry we have a pattern for this and Becca even fixed the last one 🤦 one moment

Reply:
updated and retested

Reply:
oh hmm wait i might have misunderstood. do you mean if these are logged during the creation of the flare the errors won't be visible in debugs?

Reply:
i think we're okay with this update anyway, the error should bubble up in the summary

Reply:
Oh, I was suggesting putting the errors into `powerEventFile`, but I guess that's uglier with the json format, and you're right. It is in the summary.